apps/nextjs/src/styles/globals.css

```
@@ -1,3 +1,127 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+/************************************************/
+/*                 CSS RESETS                   */
+/************************************************/
+
+/* Resets */
+html,
+body,
+#root {
+  @apply bg-black;
+  @apply font-sans;
+  @apply text-slate-50;
+  height: 100%;
+}
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-rendering: optimizeLegibility;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  letter-spacing: 0;
+}
+
+button {
+  outline: none !important;
+}
+
+/*
+ * Inputs
+ */
+::-webkit-calendar-picker-indicator {
+  filter: invert(1);
+}
+
+input:-webkit-autofill,
+input:-webkit-autofill:hover,
+input:-webkit-autofill:focus,
+textarea:-webkit-autofill,
+textarea:-webkit-autofill:hover,
+textarea:-webkit-autofill:focus,
+select:-webkit-autofill,
+select:-webkit-autofill:hover,
+select:-webkit-autofill:focus {
+  -webkit-text-fill-color: #ffffff;
+  -webkit-box-shadow: 0 0 0px 1000px rgba(37, 41, 45, 1) inset;
+  transition: background-color 5000s ease-in-out 0s;
+}
+
+/* Accesibility options */
+@media (prefers-reduced-motion: reduce) {
+  *,
+  ::before,
+  ::after {
+    animation-delay: -1ms !important;
+    animation-duration: 1ms !important;
+    animation-iteration-count: 1 !important;
+    background-attachment: initial !important;
+    scroll-behavior: auto !important;
+    transition-duration: 0s !important;
+    transition-delay: 0s !important;
+  }
+}
+
+/* class define for the usage of DottedBackground Component */
+.dotted-background {
+  background-image: radial-gradient(
+    circle,
+    var(--tw-gradient-stops) var(--dots-size),
+    var(--tw-gradient-stops) var(--dots-size),
+    transparent var(--dots-size),
+    transparent 100%
+  );
+
+  background-size: var(--dots-separation);
+  mask-image: var(--mask-image);
+}
+
+/* React Flow */
+.react-flow__controls {
+  @apply border border-slate-700;
+}
+
+.react-flow__controls .react-flow__controls-button {
+  @apply border-slate-700;
+  @apply bg-slate-950;
+}
+
+.react-flow__controls .react-flow__controls-button:hover {
+  @apply bg-slate-900;
+}
+
+.react-flow__controls .react-flow__controls-button:disabled {
+  @apply bg-slate-800;
+}
+
+.react-flow__controls .react-flow__controls-button:disabled svg {
+  @apply fill-slate-300;
+}
+
+.react-flow__controls .react-flow__controls-button svg {
+  @apply text-slate-50;
+  @apply fill-slate-50;
+}
+
+.react-flow__panel.react-flow__attribution {
+  @apply hidden;
+}
+
+/************************************************/
+/*                 UTILITY CLASSES              */
+/************************************************/
+
+@layer utilities {
+  .rotate-y-0 {
+    transform: rotateY(0);
+  }
+  .rotate-y-90 {
+    transform: rotateY(90deg);
+  }
+  .-rotate-y-90 {
+    transform: rotateY(-90deg);
+  }
+}
```

```suggestion
/************************************************/
/*                 UTILITY CLASSES              */
/************************************************/

@layer utilities {

}
```

Igual acá esta demás esta utility, ya que tenemos `tailwindcss-animated`

---

DONE